Assets/Scripts/Game/Actor.cs

```diff
@@ -5,29 +5,37 @@
 
 public class Actor : Entity
 {
-    [SerializeField] private int maxHealth;
+    [SerializeField] private int initialMaxHealth;
+
+    public int MaxHealth
+    {
+        get { return GetProperty<int>(PropertyName.MaxHealth); }
+        private set { SetProperty<int>(PropertyName.MaxHealth, value); }
+    }
 
     public int Health
     {
-        get { return GetProperty<int>(PropertyName.HEALTH); }
-        private set { SetProperty<int>(PropertyName.HEALTH, value); }
+        get { return GetProperty<int>(PropertyName.Health); }
+        private set { SetProperty<int>(PropertyName.Health, value); }
     }
```

Comment:
The arrow syntax for your properties would be a bit cleaner here

Reply:
Yeah I want to do a full codebase refactor for stuff like this. I'm planning to do that directly on `dev` after this PR is merged, as well as updating our folder structure. Let me know if there's any other convention stuff that should be changed! :)